features/FEATURE_BLE/source/BLE.cpp

```diff
@@ -191,7 +191,7 @@ void defaultSchedulingCallback(BLE::OnEventsToProcessCallbackContext* params) {
 bool BLE::hasInitialized(void) const
 {
     if (!transport) {
-        error("bad handle to underlying transport");
+        MBED_ERROR(MBED_MAKE_ERROR(MBED_MODULE_BLE, MBED_ERROR_CODE_BLE_BAD_TRANSPORT_HANDLE), "bad handle to underlying transport");
```

Comment:
Same as above, I wold use a more generic `INVALID_STATE` code or better a `NOT_INITIALIZED` error code.
I would use the same code for all errors bellow.

Reply:
The error code can be named `BLE_BACKEND_NOT_INITIALIZED`. The term _transport_ present in the code is incorrect.

Reply:
I'll fix that. Thanks.